ontology/obib.md

```diff
@@ -8,7 +8,9 @@ contact:
   github: zhengj2007
   orcid: 0000-0002-2999-0103
 description: An ontology built for annotation and modeling of biobank repository and biobanking administration
-domain: biobanking, specimens, bio-repository, biocuration
+domain: experiments
+tags:
+  - biobanking, specimens, bio-repository, biocuration
```

Comment:
different tags? also I'd suggest removing "biocuration" since this is a bit redundant

Reply:
fine by me!

Reply:
```suggestion
  - biobanking
  - specimens
  - bio-repository
```